src/Illuminate/Encryption/Encrypter.php

```diff
@@ -254,14 +254,4 @@ protected function validMac(array $payload)
             $this->hash($payload['iv'], $payload['value']), $payload['mac']
         );
     }
-
-    /**
-     * Get the encryption key.
-     *
-     * @return string
-     */
-    public function getKey()
-    {
-        return $this->key;
-    }
```

Comment:
The whole point of this PR is that this violates the contact so that's why it was removed. It's better to have it removed so that the callers have to fix their contract violations than leaving it in there and by doing so leaving a chance that the packages out there still violate the contract which makes this entire PR useless then.